starter-code/views/logIn.ejs

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Log In</title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Log In</h1>
+      <div class="">
+        <p>Log in to access your account</p>
+      </div>
+      <form id="form" class="form" action="/logIn" method="post">
+        <label for="username">Select username</label>
+        <input id="username" name="username" value="">
+
+        <label for="password">Select password</label>
+        <input id="password" name="password" value="">
+        <button type="submit" name="submit">LogIn</button>
```

Comment:
Somewhere in this template you should have a conditional block to render error messages, used when rendering the form back with an error

```
<% if (errorMessage) { %>
  <div class="error-message"><%= errorMessage %></div>
<% } %>
```